contracts/LendingPool.sol

```diff
@@ -72,7 +72,11 @@ contract LendingPool is OwnableUpgradeable, ERC721Upgradeable, Clone {
         if(toReduce > currentDailyBorrows){
             currentDailyBorrows = toAdd;
         } else {
-            currentDailyBorrows = uint216(currentDailyBorrows - toReduce) + toAdd;
+            unchecked {
+                // Resulting daily borrow has to be over 10^47 ETH
```

Comment:
this is safu because we know toReduce <= currentDailyBorrows, so (currentDailyBorrows - toReduce) >= 0